2-numpy/presentation.ipynb

```diff
@@ -506,6 +506,11 @@
     "list_ = [elem * 2 if elem % 2 else elem for elem in list_]\n",
     "\n",
     "arr = np.arange(n_elems, dtype=int)\n",
+    "mask = arr % 2 == 1\n",
+    "arr[mask] *= 2\n",
+    "\n",
+    "# Or simply\n",
+    "arr = np.arange(n_elems, dtype=int)\n",
```

Comment:
I don't think this is helpful, but if you have reason to believe that this would be helpful for the students then you should go ahead with the change.

Reply:
Fair. My intention was to make it easier or more visual to explain what is going on, but going over it again I think it neither helps nor hinders. The important part is the manual is clear. So I restored it in the new commit.